cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,403 @@
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+1. Submit the asynchronous `DELETE` request, supplying your cluster ID, API key, and the path to the certificate JSON with your CA certificate, as described in [Create the certificate authority (CA) certificate](#create-the-certificate-authority-ca-certificate).
+
+    A `200` successful response code indicates that the asynchronous request was successfully submitted, but does not guarantee that the operation (configuring the CA certificate) successfully completed. You must confirm success with a follow-up `GET` request, as described in the next step.
+ 
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request DELETE \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}"
+    ~~~
+
+    ~~~txt
+    200 OK
+    ~~~
+
+1. Confirm success of the operation with the following `GET` request.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}"
+    ~~~
+
+    `PENDING` indicates that the operation is still in process.
+    
+    ~~~txt
+    {
+      "status": "PENDING",
+      "x509_pem_cert": ""
+    }
+    ~~~
+    
+    `IS_SET` indicates that the operation completed successfully, confirming the configured public CA cert.
+
+    ~~~txt
+    {
+      "status": "IS_SET",
+      "x509_pem_cert": "-----BEGIN CERTIFICATE-----\nMIIDfzCCAmagAwIBAgIBADANBgkqhkiG9w0BAQ0FADBZMQswCQYDVQQGEwJ1czEL\nMAkGA1UECAwCV0ExDTALBgNVBAoMBHRlc3QxDTALBgNVBAMMBHRlc3QxEDAOBgNV\nBAcMB1NlYXR0bGUxDTALBgNVBAsMBHRlc3QwHhcNMjMwMzE2MjMyNTMxWhcNMjQw\n
+    ...\n-----END CERTIFICATE-----",
+    }
+    ~~~
+</section>
+<section class="filter-content" markdown="1" data-scope="tf">
+
+To delete the client CA cert on a cluster, remove the `cockroach_client_ca_cert` resource from your Terraform template, then run `terraform apply`.
+</section>
+
+## Authenticate a SQL client to a {{ site.data.products.dedicated }} cluster
+
+To use certificate authentication for a SQL client, you must include the filepaths to the client's private key and public certificate. The public certificate must be signed by a CA that the cluster has been configured to trust. Refer to [Upload a certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster](#upload-a-certificate-authority-ca-certificate-for-a-cockroachdb-dedicated-cluster).
+
+1. From your cluster's overview page, `https://cockroachlabs.cloud/cluster/{ your cluster ID }`, click the **Connect** button.
+
+1. Copy the command listed under **Download CA Cert** and run it locally to download the required public certificate, which your client will use to verify the identity of the cluster.
```

Comment:
Maybe clarify here that this CA certificate is for the cluster, to prove the server identity to the SQL client. It shouldn't be confused with the CA cert created and uploaded above.

Reply:
It says "which your client will use to verify the identity of the cluster", not sure what else to say?

Reply:
Alright, we can leave it as is.